make-rules/prep-svn.mk

```diff
@@ -48,6 +48,7 @@ CLEAN_PATHS += $$(COMPONENT_SRC$(1))
 CLOBBER_PATHS += COMPONENT_ARCHIVE$(1)
 SOURCE_DIR$(1) = $$(COMPONENT_DIR)/$$(COMPONENT_SRC$(1))
 
+download::      $$(USERLAND_ARCHIVES)
 download::	$$(USERLAND_ARCHIVES)$$(COMPONENT_ARCHIVE$(1))
```

Comment:
This is already done in prep-download.mk which is included from prep.mk which in turn is included from common.mk.
So in my eyes this looks superfluous.